assets/src/blocks/ShareButtons/FacebookShare/FacebookShareFrontend.js

```diff
@@ -0,0 +1,31 @@
+import { SvgIcon } from "../SvgIcon";
+import { parseUtmParams } from '../utils';
+
+const { __ } = wp.i18n;
+
+export const FacebookShareFrontend = ({
+  url,
+  utmMedium,
+  utmContent,
+  utmCampaign,
+  gaEvent,
+  gaEventCategory,
+  gaCategory,
+  gaAction,
+  gaLabel,
+  openInNewTab,
+  baseSharedUrl,
+}) => (
+  <a href={ `${baseSharedUrl}?u=${encodeURIComponent(url)}&${parseUtmParams('facebook', utmMedium, utmContent, utmCampaign)}` }
```

Comment:
I don't see why we would define this `baseSharedUrl` outside of this component, [in the default value of `ShareButtons` `facebook` attribute](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/758/files#diff-e0501445f43571f76b44fabd75c63eff89481ef28f79a2cfb31d203654f86c84R50). It's only ever used here, it could be a simple `const` in this file.

Reply:
Because any base shared url could change in the future, so, I think it would be better to allow to editors to edit them in case of, for example, Facebook updates its API.

Reply:
Can you give me an indication of how much you expect these URLs to change?

Reply:
> Can you give me an indication of how much you expect these URLs to change?
A while time ago I had to lead with the Instagram API that its URL changed to a FB domain. But honestly, it does not happen frequently 🙏